src/lib/optimizer/strategy/predicate_merge_rule.hpp

```diff
@@ -25,7 +25,7 @@ class PredicateMergeRule : public AbstractRule {
   void apply_to(const std::shared_ptr<AbstractLQPNode>& root) const override;
 
   // Simple heuristic: The PredicateMergeRule is more likely to improve the performance for complex LQPs.
-  size_t optimization_threshold{4};
+  size_t optimization_threshold{10}; // > TPCDS-13
```

Comment:
That name does really tell anything.

What does it do?

Reply:
I renamed it to `minimum_union_count`.